test/modules/appeals-service/AppealsService.test.ts

```diff
@@ -193,37 +235,92 @@ describe('AppealsService', () => {
 
             [undefined, null].forEach(async invalidToken => {
                 try {
-                    await appealsService.getAppeal(testCompanyNumber, testAppealId, invalidToken!);
+                    await appealsService.getAppeal(testCompanyNumber, appealId, invalidToken!, REFRESH_TOKEN);
                 } catch (err) {
                     expect(err).to.be.instanceOf(Error)
-                        .and.to.haveOwnProperty('message').equal('Token is missing');
+                        .and.to.haveOwnProperty('message').equal('Access token is missing');
+                }
+            });
+
+            [undefined, null].forEach(async refreshToken => {
+                try {
+                    await appealsService.getAppeal(testCompanyNumber, appealId, BEARER_TOKEN, refreshToken!);
+                } catch (err) {
+                    expect(err).to.be.instanceOf(Error)
+                        .and.to.haveOwnProperty('message').equal('Refresh token is missing');
                 }
             });
 
         });
 
         it('should return an appeal when valid arguments are provided', async () => {
 
-            nock(HOST)
+            nock(APPEALS_HOST)
                 .get(`${APPEALS_URI}/${APPEAL_ID}`)
                 .reply(200, appeal);
 
             const returnedAppeal = await appealsService
-                .getAppeal(appeal.penaltyIdentifier.companyNumber, APPEAL_ID, BEARER_TOKEN);
+                .getAppeal(appeal.penaltyIdentifier.companyNumber, APPEAL_ID, BEARER_TOKEN, REFRESH_TOKEN);
 
             expect(returnedAppeal).to.deep.eq(appeal);
 
         });
 
+        it('should return an appeal when valid arguments are provided after refreshing an expired token', async () => {
```

Comment:
I think its worth being as specific as possible
```suggestion
        it('should return an appeal when valid arguments are provided after refreshing an expired access token', async () => {
```